src/chunk.c

```diff
@@ -4855,3 +4861,20 @@ ts_chunk_get_osm_chunk_id(int hypertable_id)
 
 	return chunk_id;
 }
+
+/* when a tiered chunk is created, it is assigned a range [INT64_MAX -1, infinity)
+ XXX could a range start less than INT64_MAX - 1 be a valid range? */
```

Comment:
latest code will set it to that value and update all existing osm chunks to have that range but previous versions had different values for range_start for the osm chunk

Reply:
updated comment